DOCUMENTATION.md

```diff
@@ -0,0 +1,33 @@
+# DLA Future Documentation
+
+## API documentation
+
+- [Documentation of `master` branch](https://eth-cscs.github.io/DLA-Future/master/)
+- [Documentation of `v0.4.1`](https://eth-cscs.github.io/DLA-Future/v0.4.1/)
+- [Documentation of `v0.4.0`](https://eth-cscs.github.io/DLA-Future/v0.4.0/)
+- [Documentation of `v0.3.1`](https://eth-cscs.github.io/DLA-Future/v0.3.1/)
+- [Documentation of `v0.3.0`](https://eth-cscs.github.io/DLA-Future/v0.3.0/)
+- [Documentation of `v0.2.1`](https://eth-cscs.github.io/DLA-Future/v0.2.1/)
+- [Documentation of `v0.2.0`](https://eth-cscs.github.io/DLA-Future/v0.2.0/)
+- [Documentation of `v0.1.0`](https://eth-cscs.github.io/DLA-Future/v0.1.0/)
```

Comment:
Do you think it'd be enough to list the master and latest version documentation in `README.md` and leave out the links from here?